docs/docs/components/prism_highlight.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { highlightJsx } from '@mapbox/dr-ui/highlight/jsx';
```

Comment:
Probably not part of this PR but using a project that writes the following is shooting ourselves in the foot:
>This project is for internal Mapbox usage. The code is open source and we appreciate bug reports; but we will only consider feature requests and pull requests from Mapbox developers.